packages/typescript-sdk/babel.config.js

```diff
@@ -0,0 +1,40 @@
+const getPresets = () => {
```

Comment:
Nice. Im wondering if it would make sense to extract this function and share it with the root level config? Just to make it easier for future changes.

Reply:
didn't found a good extension mechanism so lets stay with that temporarily